staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -97,6 +98,8 @@ type Controller struct {
 	nodeSyncCh chan interface{}
 	// needFullSync indicates if the nodeSyncInternal will do a full node sync on all LB services.
 	needFullSync bool
+	// loadBalancerClasses indicates list of accepted LoadbalancerClasses. if empty, the provider accepts all.
+	loadBalancerClasses []*string
```

Comment:
If we're going to allow the provider implementations to allow-list the class names they should watch, it should be done through the LoadBalancer interface instead of a parameter into the controllers constructor. 

Btw, I also added this topic in the next SIG meeting in case other folks have thoughts/opinons on this.